### ISSUE TITLE
[$40][MSFT-206] Sample challenge page: Errors in the form fields in the 'challenge prerequisite' popup are being displayed using only colour.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2869]
**Steps to Reproduce**
1. Go to sample challenge page: https://www.topcoder.com/challenges/30095347. 2. Activate the register button. 3. Activate the 'continue' button, the challenge prerequisites popup is displayed. 4. Type 'abc' in all the form fields such as 'first line of address', 'second line of address' etc. 5. Notice that there are red coloured markings against these fields indicating that there are errors. There is no text or error message.

**Expected Result**
There should be error messages tagged to the respective fields.

**Actual Result**
Errors are being marked using only colour, there is no error text tagged to the form elements.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.1.1 Non-Text Content
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2035

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095982 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30095982 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by asadath1395** [comment]
@veshu @mishacucicea Can you please give a challenge ID where i can reproduce this? Thanks

**Comment by mishacucicea** [comment]
@drasticdpk Any idea what type of challenge is needed to get the additional screen/pop-up?

**Comment by veshu** [comment]
@mishacucicea This is NDA related project. If we have NDA required then this popup will come. I am trying to create the challenge with NDA but not able to run on dev.

**Comment by veshu** [comment]
@asadath1395 Please check with this challenge https://www.topcoder-dev.com/challenges/30055273?nocache=true

Challenge Id : 30055273

**Comment by asadath1395** [comment]
@veshu I am not able to get the popup with any of the given credentials

**Comment by veshu** [comment]
Yes, I am checking with it with admin team, I will get back here as soon as I get the solution

**Comment by veshu** [comment]
@asadath1395 please check with this challenge id 30055275

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096817 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by r0hit-gupta** [comment]
@veshu can we get some other challenge ID? The ones you provided do not show the NDA form for me. Please see if something can be done. Thanks!

**Comment by veshu** [comment]
Ok I will try to create another.

**Comment by veshu** [comment]
@r0hit-gupta Please check with this challenge http://localhost:3000/challenges/30055275

I used following credentials to test the registration. pat_monahan

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096817 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
@r0hit-gupta  Any progress on this one?  Thought there was a related ticket, not sure if this is still blocked or not

**Comment by r0hit-gupta** [comment]
@crazyk07 its related to DocuSign.

**Comment by crazyk07** [comment]
Closing.  Unable to fix 3rd party tool for the time being.

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
